From 36654ac05e6aad8232064b676809968364b03d10 Mon Sep 17 00:00:00 2001
From: Moritz Warning <moritzwarning@web.de>
Date: Sun, 4 Aug 2019 03:56:37 +0200
Subject: [PATCH 3/9] remove arm32 conservative CFLAGS

---
 make-linux.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/make-linux.mk b/make-linux.mk
index 7dc1a867..28fd291c 100644
--- a/make-linux.mk
+++ b/make-linux.mk
@@ -272,7 +272,7 @@ ifeq ($(ZT_CONTROLLER),1)
 endif
 
 # ARM32 hell -- use conservative CFLAGS
-ifeq ($(ZT_ARCHITECTURE),3)
+ifeq (0,3)
 	ifeq ($(shell if [ -e /usr/bin/dpkg ]; then dpkg --print-architecture; fi),armel)
 		override CFLAGS+=-march=armv5t -mfloat-abi=soft -msoft-float -mno-unaligned-access -marm
 		override CXXFLAGS+=-march=armv5t -mfloat-abi=soft -msoft-float -mno-unaligned-access -marm
-- 
2.31.1

